src/oauth2-client/index.ts

```diff
@@ -25,6 +25,8 @@ abstract class OAuth2Client {
    * Gets the authorization URL configured in OpenAM, optionally using PKCE.
    */
   public static async getAuthorizeUrl(options: GetAuthorizationUrlOptions): Promise<string> {
+    console.warn('Deprecation warning: this `getAuthorizeUrl` method will be renamed in v3.');
```

Comment:
I remember we were talking about renaming this method, but forgot why exactly?

Reply:
It's because the method's name is misleading. This method doesn't actually return the authorize URL, it returns the callback URL that results from the authorize call :)

Reply:
I see... Makes sense then!